src/shared/lightningDevConfig.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ConfigFile } from '@salesforce/core';
+
+export const LIGHTNING_DEV_CONFIG = 'lightningDevConfig.json';
+
+export class LightningDevConfig extends ConfigFile {
+  public static getFileName(): string {
+    return LIGHTNING_DEV_CONFIG;
+  }
+}
```

Comment:
I have this class because I couldn't get the default config file to work.

Default config is a file named `config.json` that lives in sfdx project's `.sf` folder. The properties can be read from there, however, not be written. When I try to write I get the error as shown in the screen shot. This error doesn't happen if I specify a different file.
![image](https://github.com/salesforcecli/plugin-lightning-dev/assets/42476999/8ecbbfa9-7b06-4aac-9b7b-cc53c275620a)

Reply:
This is what I am doing, @mshanemc